### ISSUE TITLE
Feature Request: enable customization of '--format-sort' 

### ISSUE THREAD

**Issue by cxumol** [tropicoo/yt-dlp-bot#183]
In the latest updates, video codec option is freeze to h265, h264 at `app_worker/ytdl_opts/per_host/_base.py`. Usually YouTube provided better video quality on vp9(.webm) or av1 on high resolution pieces; Some user's device may not be compatible with h265. Therefore I believe the '--format-sort' is reasonable to be customizable on the user side.

**Comment by tropicoo** [comment]
Yep, agree. I believe I forgot to add that as a configuration variable.

**Comment by tropicoo** [comment]
I've put the variable to the `default.py` which can be overridden in the `user.py`.
Please reopen if needed
